worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -116,7 +118,7 @@ public void sphereBrush(Player player, LocalSession session,
             tool.setBrush(new SphereBrush(), "worldedit.brush.sphere");
         }
 
-        player.print(String.format("Sphere brush shape equipped (%.0f).", radius));
+        player.printInfo(TranslatableComponent.of("worldedit.brush.sphere.equip", TextComponent.of((int) radius)));
```

Comment:
Did you intend to lose precision here? Old code could format a one-digit decimal as well, but this is just integers.